### DISCUSSION TITLE
Font request: KA-Almouzios

### DISCUSSION THREAD

**Discussion by danielgarthur** [neanes/neanes#868] [Ideas]
Requested on behalf of @ilizol. 

https://github.com/t-bullock/KA-Almouzios

**Comment by t-bullock** [comment]
I don't have the time to modify the font for Neanes, but I did upload the sources in .ufo format. I'd love to see this font incorporated into Neanes as well!

**Reply by t-bullock** [reply]
Also just wanted to add, if there's anything else I can do to help someone else get this font converted, please let me know. Whoever does take this up, feel free to change the name of the font. I called it "Almouzios" at the time as a joke because it looks almost like the popular Chourmouzios font, and I didn't take the time to think up a good name for it.

**Reply by ilizol** [reply]
WORK IN PROGRESS
https://github.com/ilizol/Almouzios

Please check if the new name "Almouzios" and the OFL-1.1 license are acceptable.
Thanks for your patience in advance.
